database/service_db_test.go

```diff
@@ -82,7 +84,10 @@ func TestServiceDBDelete(t *testing.T) {
 	require.IsType(t, &ErrNotFound{}, err)
 }
 
+// TOFIX: the database is not thread safe anymore...
```

Comment:
the database is still thread safe, but the question is why the transaction has gone

Reply:
kvstore does not implement transaction as it should modify and read the store one transaction at a time. so they don't need concurrent access..